man/systemd-system.conf.xml

```diff
@@ -225,6 +225,15 @@
         understood too.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ShortIdentifiers=</varname></term>
```

Comment:
hmm, "ShortIdentifiers" might be a bit short an identifier. probably should mention "unit" and "status" in the setting's name I think....

UseUnitNameInStatus=?
StatusShortUnitIdentifiers=?
CompactUnitStatus=?
UnitStatusCompactNames=?

something like that?

Reply:
Also, I wonder if this maybe should be an enum: "StatusUnitIdentifiers=name|description|both" or so? And in the "both" case we'd generate "Long Description (unit-name.service)" as strings?

Reply:
Hmm, what about UnitNames=name|description|both (and `systemd.unit-names`, `systemd-unit-names=name|description|both`) ?  Using name as both the option name and value is not ideal, but it seems better than the other proposals to me.

Reply:
Maybe StatusUnitName=? i.e. I really think we should include a reference to "status", since this is really about the status output, i.e. the stuff controlled with ShowStatus= which already contains the "status" in the name